README.md

```diff
@@ -51,9 +51,9 @@ below.
 
 ```bash
 # switches to your home directory
-cd
+cd %userprofile%
 # activates the conda environment you installed
-datafest\Scripts\activate.bat
+datafest\Scripts\activate
```

Comment:
I actually have both, my prompt autocompleted to `activate` so I didn't see `activate.bat`.